lib/fizzy/execute.cpp

```diff
@@ -486,7 +486,7 @@ inline bool invoke_function(
 {
     const auto num_args = func_type.inputs.size();
     assert(stack.size() >= num_args);
-    span<const Value> call_args{stack.rend() - num_args, num_args};
+    const auto call_args = stack.rend() - num_args;
```

Comment:
Didn't I do the same change in some PR already? Perhaps one of your old refactorings.